libnavigation-core/src/test/java/com/mapbox/navigation/core/replay/history/MemoryTestRule.kt

```diff
@@ -0,0 +1,28 @@
+package com.mapbox.navigation.core.replay.history
+
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+
+class MemoryTestRule : TestRule {
```

Comment:
Happy to move this to a common module. Would like it to be used in [benchmarks](https://github.com/mapbox/mapbox-navigation-android/pull/4239)